### PR TITLE
Fix app not starting in Windows (10)

### PR DESCRIPTION
**Pull request by apedroferreira**
Seems to fix the issue I had in https://github.com/mui/mui-toolpad/issues/2431.

Followed the top answer from https://stackoverflow.com/questions/75883720/504-outdated-optimize-dep-while-using-react-vite.

Not 100% sure if this is the same issue the users had, but I could reproduce it every time, just the first time I ran a new project.
Also I don't have Windows 11 to be able to test there if there are any additional issues.

### CODE REVIEW

File: packages/toolpad-app/src/server/toolpadAppBuilder.ts [hunk 1]
```diff
@@ -285,6 +285,7 @@ export function createViteConfig({
           'superjson',
           'zod',
         ],
+        exclude: ['@mui_toolpad_runtime', '@mui_toolpad_canvas'],
```
Comment (Janpot) [inline packages/toolpad-app/src/server/toolpadAppBuilder.ts:288]: Does this only work with
```
@mui_toolpad_runtime
```
or also with
```
@mui/toolpad/runtime
```
?

Comment (apedroferreira) [reply packages/toolpad-app/src/server/toolpadAppBuilder.ts:288]: I can try `@mui/toolpad/runtime` in case it works.

**Comment by apedroferreira** [comment]
I'm actually not sure if this works, the issue only seems to be reproducible if the project uses a remote version of  `@mui/toolpad` such as 0.1.22.

When trying to use local changes the issue does not seem to happen which I think mislead me to think the above solution was working...
So not sure how to test fixes for this besides creating a new release with changes which is not ideal...

**Comment by Janpot** [comment]
> So not sure how to test fixes for this besides creating a new release with changes which is not ideal

does it happen when using the packages built for a PR by codesandboxci?

**Comment by apedroferreira** [comment]
> does it happen when using the packages built for a PR by codesandboxci?

Good idea, I tried and the issue is still there in this PR, at least looks like I can debug it now.

**Comment by apedroferreira** [comment]
Looks like using `@mui/toolpad/runtime` and `@mui/toolpad/canvas` is the actual way to do it, seems to be working now with the Codesandbox packages!

**Pull request merged**